tests/yam/agama/agama_grub2.pm

```diff
@@ -0,0 +1,34 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Processing grub2 after installation finishes and reboot occurs
+# integration test from GitHub.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+
+sub agama_grub2 {
```

Comment:
you don't need a function on the test layer in general, please include this in the `run` one.

Reply:
> you don't need a function on the test layer in general, please include this in the `run` one.

Changed.